src/main/java/address/sync/cloud/CloudRateLimitStatus.java

```diff
@@ -48,8 +48,9 @@ private ZoneOffset getSystemTimezone() {
 
     private long getNextResetTime() {
         LocalDateTime curTime = LocalDateTime.now();
+        LocalDateTime nextHour = curTime.plusHours(1);
```

Comment:
Sure.
On Jun 29, 2016 12:26 AM, "Martin Choo" <notifications@github.com> wrote:

> In src/main/java/address/sync/cloud/CloudRateLimitStatus.java
> <https://github.com/HubTurbo/addressbook/pull/329#discussion_r68791913>:
>
> > @@ -48,8 +48,9 @@ private ZoneOffset getSystemTimezone() {
> >
> >      private long getNextResetTime() {
> >          LocalDateTime curTime = LocalDateTime.now();
> > +        LocalDateTime nextHour = curTime.plusHours(1);
>
> Think can just call LocalDateTime.now().plusHours(1)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/HubTurbo/addressbook/pull/329/files/945d577c2cc83122109a35b96861571eb59fae57#r68791913>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ADOZIIe-rxZl3zQnaDjgjY9SkSMRxzQzks5qQUtPgaJpZM4JAQJe>
> .
>